backend/migrations/versions/d65d4263d860_add_date_assigned_to_usergoal_primary_.py

```
@@ -0,0 +1,67 @@
+from alembic import op
+import sqlalchemy as sa
+
+
+# revision identifiers, used by Alembic.
+revision = "d65d4263d860"
+down_revision = "cf27bcdfcca9"
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
```

Ruff (0.8.0)

Missing return type annotation for public function `upgrade`

Add return type annotation: `None`

(ANN201)